SRTM6000/srtm_45_13.hdr

```diff
@@ -0,0 +1,32 @@
+Geotiff_Information:
+   Version: 1
+   Key_Revision: 1.0
+   Tagged_Information:
+      ModelTiepointTag (2,3):
+         0                0                0                
+         40               0                0                
+      ModelPixelScaleTag (1,3):
+         0.000833333333   0.000833333333   0                
+      End_Of_Tags.
+   Keyed_Information:
+      GTModelTypeGeoKey (Short,1): ModelTypeGeographic
+      GTRasterTypeGeoKey (Short,1): RasterPixelIsArea
+      GeographicTypeGeoKey (Short,1): GCS_WGS_84
+      GeogCitationGeoKey (Ascii,7): "WGS 84"
+      GeogAngularUnitsGeoKey (Short,1): Angular_Degree
+      End_Of_Keys.
+   End_Of_Geotiff.
+
+GCS: 4326/WGS 84
+Datum: 6326/World Geodetic System 1984
+Ellipsoid: 7030/WGS 84 (6378137.00,6356752.31)
+Prime Meridian: 8901/Greenwich (0.000000/  0d 0' 0.00"E)
+
+PROJ.4 Definition: +proj=latlong +ellps=WGS84 +to_meter=1.0000000000
+
+Corner Coordinates:
+Upper Left    (40.0000000,0.0000000)
+Lower Left    (40.0000000,-5.0000000)
+Upper Right   (45.0000000,0.0000000)
+Lower Right   (45.0000000,-5.0000000)
+Center        (42.5000000,-2.5000000)

```

Comment:
The files in this folder I believe are not needed in the main branch. Also don't upload binaries like srtm2sdf

Reply:
DONE, removed.